docs/administering_lagoon/rbac.md

```diff
@@ -136,6 +136,8 @@ Here is a table that lists the roles and the access they have:
 | taskDrushSqlDump                          | task         | drushSqlDump:production              | projectID  | Yes                 | Yes                 | Yes   | Yes        | Yes       | No       | No    |      |
 | taskDrushCacheClear                       | task         | drushCacheClear:development          | projectID  | Yes                 | Yes                 | Yes   | Yes        | Yes       | Yes      | Yes   |      |
 | taskDrushCacheClear                       | task         | drushCacheClear:production           | projectID  | Yes                 | Yes                 | Yes   | Yes        | Yes       | Yes      | Yes   |      |
+| taskDrushCron                             | task         | taskDrushCron:development            | projectID  | Yes                 | Yes                 | Yes   | Yes        | Yes       | Yes      | Yes   |      |
```

Comment:
I just caught that the scope names here don't match in code, shouldn't have the `task` prefix.